.github/workflows/rails.yml

```
@@ -0,0 +1,56 @@
+# For more information see:
+# - https://docs.github.com/en/actions/guides/building-and-testing-ruby
+# - https://github.com/niclin/rails-github-actions-example/blob/master/.github/workflows/rails.yml
+
+name: Rails
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

```suggestion
    branches: [ master ]
  pull_request:
    branches: [ master ]
```
(possibly we should change our main branch name but that's out of scope)

---

\o/